submissions/UA-girl/Task-JS-DOM/js/main.js

```diff
@@ -0,0 +1,77 @@
+(function () {
+  const menu = document.querySelector('.navigation-list');
+  const menuButton = document.querySelector('.navigation-button');
+  const form = document.querySelector('.booking-form');
+  const table = document.querySelector('.order-table');
+  let order = {
+    city: 'Київ',
+    date: '2021-01-30',
+    time: '10:00',
+    place: [],
+    total: 0,
+
+    dateString: function () {
+      const [year, month, date] = this.date.split('-');
+      return `${date}.${month}.${year}p.`;
+    },
+
+    placeString: function () {
+      return this.place
+        .map((p) => {
+          const [row, seat] = p.value.split(' ');
+          return `${row} ряд ${seat} місце`;
+        })
+        .join('; ');
+    },
+
+    totalString: function () {
+      this.total = 0;
+      this.place.forEach((p) => {
+        this.total += +p.dataset.price;
+      });
+      return `${this.total}грн.`;
+    },
+  };
+
+  function updateTable(name, value) {
+    table.querySelector(`.${name}`).textContent = value;
+  }
+
+  function inputHandler(e) {
+    e.preventDefault();
+    const { name, value } = e.target;
+    let newValue, newTotal;
+    switch (name) {
```

Comment:
Switch for two cases is overkill. It's fine but simple `if else` will be enough.